src/client/src/containers/DependencyInfo/index.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import styles from "./styles.module.css";
+import * as getSvg from "../../utils/getSvgUrl";
+import classnames from "classnames";
+import { injectIntl, defineMessages } from "react-intl";
+import { WIZARD_CONTENT_INTERNAL_NAMES } from "../../utils/constants";
+import { AppState } from "../../reducers";
+import { IDependenciesInstalled } from "../../reducers/dependencyInfoReducers";
+
+interface IDependencyInfoProps {
+  frameworkName: string;
+  intl: any;
+  dependenciesStore: IDependenciesInstalled;
+}
+
+type Props = IDependencyInfoProps;
+
+const messages = defineMessages({
+  installed: {
+    id: "dependencyChecker.installed",
+    defaultMessage: " detected!"
+  },
+  notInstalled: {
+    id: "dependencyChecker.notInstalled",
+    defaultMessage: " not detected. Click to install."
+  },
+  iconAltMessage: {
+    id: "dependencyChecker.iconAltMessage",
+    defaultMessage: "Icon for dependency checker"
+  }
+});
+
+interface IDependency {
+  commandName: string;
+  dependencyName: string;
+  downloadLink: string;
+  privacyStatementLink: string;
+}
+
+interface IDependencies {
+  [key: string]: IDependency;
+}
+
+const dependencies: IDependencies = {
+  NodeJS: {
+    commandName: "node", // command used on the extension side
+    dependencyName: "Node",
+    downloadLink: "https://nodejs.org/en/download/",
+    privacyStatementLink: "https://nodejs.org/en/about/privacy/"
+  },
+  Python: {
+    commandName: "python",
+    dependencyName: "Python",
+    downloadLink: "https://www.python.org/downloads/",
+    privacyStatementLink: "https://www.python.org/privacy/"
+  }
+};
+
+const frameworkNameToDependencyMap: Map<string, IDependency> = new Map([
+  [WIZARD_CONTENT_INTERNAL_NAMES.ANGULAR, dependencies.NodeJS],
+  [WIZARD_CONTENT_INTERNAL_NAMES.FLASK, dependencies.Python],
+  [WIZARD_CONTENT_INTERNAL_NAMES.REACT_JS, dependencies.NodeJS],
+  [WIZARD_CONTENT_INTERNAL_NAMES.VUE, dependencies.NodeJS],
+  [WIZARD_CONTENT_INTERNAL_NAMES.NODE_JS, dependencies.NodeJS]
+]);
+
+/*
+ * Props:
+ * - frameworkName: string
+ */
+class DependencyInfo extends React.Component<Props> {
+  public render() {
+    let { frameworkName, intl, dependenciesStore } = this.props;
+    let dependency: IDependency | undefined = frameworkNameToDependencyMap.get(
+      frameworkName
+    );
+
+    if (dependency === undefined) {
+      return null; // don't render anything
+    }
+
+    const { dependencyName, downloadLink, privacyStatementLink } = dependency;
```

Comment:
Just wondering where is the privacyStatementLink used? also you can add commandName here too, makes the code a bit cleaner.

Reply:
It's not used atm, fixed!